### PR TITLE
Hide signup button if user is logged in.

### PR DESCRIPTION
**Pull request by SG-coder**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1101/120640.574:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (silwalanish) [inline debug.log:1]: Delete this file.

File: src/Pages/Home.jsx [hunk 2]
```diff
@@ -24,13 +25,19 @@ class Home extends React.Component {
...
-              <button
-                type='button'
-                className='btn btn btn-primary'
-                onClick={this.props.signup}
-              >
-                Signup
-              </button>
+              {this.props.isLoggedIn ? (
+                <span>
+                  <Link to={EXPLORE_URL}>explore</Link>
```
Comment (silwalanish) [inline src/Pages/Home.jsx:30]:
```suggestion
                  <Link to={EXPLORE_URL}>Explore</Link>
```

**Comment by silwalanish** [comment]
@SG-coder Remove `debug.log` file and add the screenshot of the home page (before and after login) in the PR description. Looks good to me.
cc: @KGhising @danny237

File: src/Pages/Home.jsx [hunk 3]
```diff
@@ -24,13 +25,23 @@ class Home extends React.Component {
...
-                type='button'
-                className='btn btn btn-primary'
-                onClick={this.props.signup}
-              >
-                Signup
-              </button>
+              {this.props.isLoggedIn ? (
+                <span>
+                  <Link to={EXPLORE_URL}>
+                    <button type='button' className='btn btn btn-primary'>
```
Comment (silwalanish) [inline src/Pages/Home.jsx:31]:
```suggestion
                    <button type='button' className='btn btn-primary'>
```

File: src/Pages/Home.jsx [hunk 4]
```diff
@@ -24,13 +25,23 @@ class Home extends React.Component {
...
+                      Explore
+                    </button>
+                  </Link>
+                </span>
+              ) : (
+                <button
+                  type='button'
+                  className='btn btn btn-primary'
```
Comment (silwalanish) [inline src/Pages/Home.jsx:39]:
```suggestion
                  className='btn btn-primary'
```

File: src/Pages/Home.jsx [hunk 5]
```diff
@@ -24,13 +25,23 @@ class Home extends React.Component {
...
-                Signup
-              </button>
+              {this.props.isLoggedIn ? (
+                <span>
+                  <Link to={EXPLORE_URL}>
+                    <button type='button' className='btn btn-primary'>
+                      Explore
+                    </button>
+                  </Link>
+                </span>
```
Comment (silwalanish) [inline src/Pages/Home.jsx:35]:
```suggestion
                <Link to={EXPLORE_URL}>
                  <button type='button' className='btn btn-primary'>
                    Explore
                  </button>
                </Link>
```

**Review by silwalanish — APPROVED** [review]

**Review by danny237 — APPROVED** [review]

**Review by KGhising — APPROVED** [review]

**Pull request merged**